types/transform.go

```diff
@@ -189,3 +308,29 @@ func (t *Transform) ParseAsWriteDeployInfo() (*DeployInfo, error) {
 
 	return &jsonRes.WriteDeployInfo, nil
 }
+
+func (t *TransformKind) ParseAsWriteCLValue() (*Argument, error) {
```

Comment:
There is a mix of `ParseAsWriteXXX` instead of `ParseAsXXX`?

Why does this method return an Argument instead of a CLValue?